tests/Nox.Reference.Data.World.Tests/CountriesInfoTests.cs

```diff
@@ -0,0 +1,42 @@
+//using System.Diagnostics;
+//using System.Text.Json;
+
+//namespace Nox.Reference.Data.World.Tests;
+
+//public class CountryInfoTests
```

Comment:
why comment?

Reply:
It doesn't make sense for now, because we should implement persisting and reading Countries in dataContext.
Service projects like: MacAddress,Currencies, Languages, Countries have been removed from solution

Reply:
maybe delete or add TODO then?